src/lib/mina_base/account_update.ml

```diff
@@ -1479,14 +1435,21 @@ module T = struct
       =
     { authorization; body = Body.to_graphql_repr ~call_depth body }
 
-  let gen caller : t Quickcheck.Generator.t =
+  let gen : t Quickcheck.Generator.t =
     let open Quickcheck.Generator.Let_syntax in
-    let%map body = Body.gen caller
-    and authorization = Control.gen_with_dummies in
+    let%map body = Body.gen and authorization = Control.gen_with_dummies in
     { body; authorization }
 
-  let to_wire (p : t) caller : Wire.t =
-    { body = Body.to_wire p.body caller; authorization = p.authorization }
+  let quickcheck_generator : t Quickcheck.Generator.t = gen
+
+  let quickcheck_observer : t Quickcheck.Observer.t =
+    Quickcheck.Observer.of_hash (module Stable.Latest)
```

Comment:
`of_hash` only needs `hash` and `hash_fold`, not any `Bin_prot` functions

but this is OK, I suppose